ide/api/phone.py

```diff
@@ -22,14 +23,18 @@ def list_phones(request):
         params={'client_id': settings.SOCIAL_AUTH_PEBBLE_KEY})
 
     if response.status_code != 200:
-        return json_failure(response.reason)
+        if 400 <= response.status_code < 500:
```

Comment:
Why are you making this distinction? It's not the client that made a bad request here.